add message if use has not coin

yukari-n-erb opened the pull request:
## Description
If use has not coin, app show message "To purchase flowers, let grew up flower and get coin!".
(for avoiding confusing for the user to understand why everything is disabled)

## Closes issue(s)

https://github.com/ice-cream-the-fat-cat/my-niwa-frontend/issues/158

## How to test / reproduce

## Screenshots

### When user has not coin
<img width="421" alt="Screen Shot 2021-09-11 at 13 26 23" src="https://user-images.githubusercontent.com/45537257/132936178-fd193fe1-9670-4cbc-b9c4-a7b7e0b85655.png">


### When user has coins
<img width="421" alt="Screen Shot 2021-09-11 at 13 27 04" src="https://user-images.githubusercontent.com/45537257/132936180-5f0e301c-4459-4ee0-bd86-e5abeac339c1.png">
<img width="421" alt="Screen Shot 2021-09-11 at 13 26 59" src="https://user-images.githubusercontent.com/45537257/132936189-8cb18b8c-d4bc-4065-ab66-6f36b97f5683.png">


## Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

## Checklist

- [x] I have tested this code

conalli commented on src/pages/Florist/index.tsx:
```diff
@@ -180,12 +180,16 @@ export const Florist = () => {
+              <Typography variant="h6">Welcome to our store.</Typography>
+              {!userData.balance ? (
+                <Typography variant="body1" className={classes.welcomeText}>
+                  To purchase flowers, let grew up flower and get coin!
```
The text could be something like "To purchase flowers you need coins! Plant some seeds and your wallet will grow."

conalli commented on src/pages/Florist/index.tsx:
```diff
@@ -56,13 +56,13 @@ const useStyles = makeStyles((theme: Theme) =>
       marginTop: "5%",
-      fontWeight: "bold"
+      fontWeight: "bold",
     },
```
Maybe add an object here for if user balance is 0 with the same styling as welcomeText but also include "backgroundColor: theme.palette.error.light"

conalli reviewed (changes requested):
Good job!

curtisahuang reviewed (approved):
lgtm!

yukari-n-erb commented:
Thank you so much! I fixed message and added background like this 🙏

![Screen Shot 2021-09-11 at 14 32 44](https://user-images.githubusercontent.com/45537257/132937478-83659c0a-3089-40fe-acfa-90b1c5fe014d.png)

kome12 commented:
> Thank you so much! I fixed message and added background like this 🙏
> 
> ![Screen Shot 2021-09-11 at 14 32 44](https://user-images.githubusercontent.com/45537257/132937478-83659c0a-3089-40fe-acfa-90b1c5fe014d.png)

@yukari-n-erb I think this is okay for now but I created another issue #185
